source/gui/settingsDialogs.py

```diff
@@ -2928,6 +2941,10 @@ def onSave(self):
 class DocumentNavigationPanel(SettingsPanel):
 	# Translators: This is the label for the document navigation settings panel.
 	title = _("Document Navigation")
+	panelDescription = _(
+		# Translators: This is descriptive text shown at the top of the Document Navigation settings panel.
+		"Configure certain specifics of how you can move through documents.",
```

Comment:
```suggestion
		"Configure the behaviour of navigation commands in documents.",
```

Not: I expect configuration for sentence navigation commands (expected during 2025 dev cycle) and word navigation commmands (if @mltony or anyone else integrate this in NVDA) to land in this panel.